src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDriver.java

```diff
@@ -257,6 +257,43 @@ static SSLProtocol valueOfString(String value) throws SQLServerException {
 }
 
 
+enum IPvAddressPreferenceEnum {
```

Comment:
remove "Enum", please see how others are named and and be consistent

Reply:
Changed.